### ISSUE TITLE
Is there anything wrong with Triplet loss script in README.md?

### ISSUE THREAD

**Issue by zhangyubing** [deepinsight/insightface#165]
There is one command line in README.md as below:

------------------------------
(4). Fine-turn the above Softmax model with Triplet loss.
        CUDA_VISIBLE_DEVICES='0,1,2,3' python -u train_softmax.py --network m1 --loss-type 12 --lr 0.005 --mom 0.0 --per-batch-size 150 --data-dir ../datasets/faces_ms1m_112x112 --pretrained ../model-m1-softmax,50 --prefix ../model-m1-triplet
------------------------------

I cannot find triplet loss definition in src/train_softmax.py but find "elif args.loss_type==12: #triplet loss" in src/train.py. Should the python file be train.py instead of train_softmax.py in README.md for triplet loss?

**Comment by s0302102** [comment]
the preivous train_softmax.py supported tripletloss. The author deleted those codes during a code-cleaning process. I guess you can find it from the previous commits

**Comment by nttstar** [comment]
Yes, triplet loss is now in `train.py` and `data.py`. `train_softmax.py` and 'image_iter.py` are more clean.

**Comment by zhangyubing** [comment]
THX!
